internal/clientconn/conn.go

```diff
@@ -441,6 +442,24 @@ func (c *conn) route(connCtx context.Context, reqHeader *wire.MsgHeader, reqBody
 
 		command = document.Command()
 
+		if comment, cerr := document.Get("comment"); cerr == nil {
```

Comment:
```go
var comment any
if comment, err = …
```

Otherwise, it is easy to use `err` instead of `cerr`, just like below

Reply:
It's not really possible because a few lines above, there is `err` returned from `msg.Dcoument`, and there are two checks of this `err` below.